### PR TITLE
Add OCP annotation to OKD bundle

### PR DESCRIPTION
**Pull request by clobrano**
#### Why we need this PR

OKD community pipeline is verifying the presence of OCP annotation.

#### Changes made

Add OCP annotation to OKD bundle

### CODE REVIEW

**Comment by beekhof** [comment]
Looks sane to me, do we understand all the annotations?
We should also make sure there is a backlog item for being FIPS compliant

File: Makefile [hunk 1]
```diff
@@ -329,9 +338,25 @@ verify-previous-version: ## Verifies that PREVIOUS_VERSION variable is set
 .PHONY: bundle-community-rh
 bundle-community-rh: bundle ## Update displayName field in the bundle's CSV
 	sed -r -i "s|displayName: Machine Deletion Remediation operator.*|displayName: Machine Deletion Remediation Operator - Community Edition|;" ${CSV}
+	$(MAKE) add-replaces-field
```
Comment (slintes) [inline Makefile:341]: I don't find this target?

Comment (clobrano) [reply Makefile:341]: Doh, sorry, it's a leftover.
MDR only have 1 bundle (OKD) that needs the replaces field, so there's no need to specialize this part

**Comment by clobrano** [comment]
> Looks sane to me, do we understand all the annotations? We should also make sure there is a backlog item for being FIPS compliant

we have ECOPROJECT-1972 to track the task, and there is a dedicated documentation page for the annotation

**Review by slintes — APPROVED** [review]

**Pull request merged**